libcarina.go

```diff
@@ -442,3 +447,19 @@ func (c *ClusterClient) Delete(clusterName string) (*Cluster, error) {
 	resp, err := c.NewRequest("DELETE", uri, nil)
 	return clusterFromResponse(resp, err)
 }
+
+func (c *ClusterClient) GetQuotas() (*Quotas, error) {
+	uri := path.Join("/quotas", c.Username)
+	resp, err := c.NewRequest("GET", uri, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	quotas := new(Quotas)
```

Comment:
I didn't do this because it seemed unnecessary. I know `clusterFromResponse` is used in several places, but this is pretty specific to just getting quotas.